### PR TITLE
docs: remove duplicated step in Build .js doc

### PR DESCRIPTION
**Pull request by ramasilveyra**
Remove duplicated step in Build OpenCV.js doc: https://docs.opencv.org/4.6.0/d4/da1/tutorial_js_setup.html

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [x] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: doc/js_tutorials/js_setup/js_setup/js_setup.markdown [hunk 1]
```diff
@@ -138,13 +138,6 @@ Building OpenCV.js from Source
     python ./platforms/js/build_js.py build_js --cmake_option="-DOPENCV_EXTRA_MODULES_PATH=opencv_contrib/modules"
     @endcode
 
--#  [optional] To enable OpenCV contrib modules append `--cmake_option="-DOPENCV_EXTRA_MODULES_PATH=/path/to/opencv_contrib/modules/"`
```
Comment (alalek) [inline doc/js_tutorials/js_setup/js_setup/js_setup.markdown:141]: Nice catch!

relates #19167

Please add `emcmake` prefix to the command above: `emcmake python ...`

Comment (ramasilveyra) [reply doc/js_tutorials/js_setup/js_setup/js_setup.markdown:141]: done!

**Comment by ramasilveyra** [comment]
@alalek this bug is also present in v5 https://docs.opencv.org/5.x/d4/da1/tutorial_js_setup.html, should I create a new pr for the 5.x branch?

**Review by alalek — APPROVED** [review]
Thank you :+1:

Changes will be propagated to 5.x branch automatically: https://github.com/opencv/opencv/wiki/Branches
Separate patch is not needed.

**Pull request merged**